### PR TITLE
[zh-cn] Resync kubeadm init page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -294,18 +295,18 @@ List of feature gates:
...
 `RootlessControlPlane` | `false` | 1.22 | -
 `UnversionedKubeletConfigMap` | `true` | 1.22 | 1.23
 {{< /table >}}
 
+{{< note >}}
 <!-- 
 Once a feature gate goes GA it is removed from this list as its value becomes locked to `true` by default. 
 -->
-{{< note >}}
 一旦特性门控变成了 GA，那它将会从这个列表中移除，因为它的值会被默认锁定为 `true` 。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:310]:
```suggestion
一旦特性门控变成了 GA，那它将会从这个列表中移除，因为它的值会被默认锁定为 `true`。
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 2]
```diff
@@ -434,43 +432,41 @@ kubeadm 需要的所有默认 `k8s.gcr.io` 镜像都支持多种硬件体系结
...
-### 使用自定义的镜像 {#custom-images}
 
-<!--
 By default, kubeadm pulls images from `k8s.gcr.io`. If the
 requested Kubernetes version is a CI label (such as `ci/latest`)
 `gcr.io/k8s-staging-ci-images` is used.
 -->
+### 使用自定义的镜像 {#custom-images}
+
 默认情况下, kubeadm 会从 `k8s.gcr.io` 仓库拉取镜像。如果请求的 Kubernetes 版本是 CI 标签
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:442]:
```suggestion
默认情况下，kubeadm 会从 `k8s.gcr.io` 仓库拉取镜像。如果请求的 Kubernetes 版本是 CI 标签
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 3]
```diff
@@ -522,100 +534,102 @@ kubeadm init phase upload-certs --upload-certs --certificate-key=SOME_VALUE --co
...
-### 设置节点的名称
 
-<!--
 By default, `kubeadm` assigns a node name based on a machine's host address. You can override this setting with the `-node-name` flag.
 The flag passes the appropriate [`-hostname-override`](/docs/reference/command-line-tools-reference/kubelet/#options)
 value to the kubelet.
 -->
+### 设置节点的名称
+
 默认情况下, `kubeadm` 基于机器的主机地址分配一个节点名称。你可以使用 `--node-name` 参数覆盖此设置。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:599]:
```suggestion
默认情况下，`kubeadm` 基于机器的主机地址分配一个节点名称。你可以使用 `--node-name` 参数覆盖此设置。
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 4]
```diff
@@ -488,23 +484,39 @@ for etcd and CoreDNS.
...
+### Uploading control-plane certificates to the cluster
+
 By adding the flag `-upload-certs` to `kubeadm init` you can temporary upload
 the control-plane certificates to a Secret in the cluster. Please note that this Secret
 will expire automatically after 2 hours. The certificates are encrypted using
 a 32byte key that can be specified using `-certificate-key`. The same key can be used
 to download the certificates when additional control-plane nodes are joining, by passing
 `-control-plane` and `-certificate-key` to `kubeadm join`.
 -->
+### 将控制平面证书上传到集群
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:518]:
```suggestion
### 将控制平面证书上传到集群   {#uploading-control-plane-certificates-to-the-cluster}
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 5]
```diff
@@ -522,100 +534,102 @@ kubeadm init phase upload-certs --upload-certs --certificate-key=SOME_VALUE --co
...
 
-<!--  
 For detailed information on certificate management with kubeadm see
 [Certificate Management with kubeadm](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/).
 The document includes information about using external CA, custom certificates
 and certificate renewal.
 -->
-有关使用 kubeadm 进行证书管理的详细信息，请参阅
-[使用 kubeadm 进行证书管理](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/)。
+### 使用 kubeadm 管理证书
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:557]:
```suggestion
### 使用 kubeadm 管理证书   {#certificate-management-with-kubeadm}
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 6]
```diff
@@ -522,100 +534,102 @@ kubeadm init phase upload-certs --upload-certs --certificate-key=SOME_VALUE --co
...
 <!--
 ### Use kubeadm with CRI runtimes
--->
-### 结合 CRI 运行时使用 kubeadm
 
-<!--
 By default kubeadm attempts to detect your container runtime. For more details on this detection, see
 the [kubeadm CRI installation guide](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime).
 -->
+### 结合 CRI 运行时使用 kubeadm
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:585]:
```suggestion
### 结合 CRI 运行时使用 kubeadm   {#use-kubeadm-with-cri-runtimes}
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 7]
```diff
@@ -522,100 +534,102 @@ kubeadm init phase upload-certs --upload-certs --certificate-key=SOME_VALUE --co
...
 ### Setting the node name
--->
-### 设置节点的名称
 
-<!--
 By default, `kubeadm` assigns a node name based on a machine's host address. You can override this setting with the `-node-name` flag.
 The flag passes the appropriate [`-hostname-override`](/docs/reference/command-line-tools-reference/kubelet/#options)
 value to the kubelet.
 -->
+### 设置节点的名称
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:597]:
```suggestion
### 设置节点的名称   {#setting-the-node-name}
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 8]
```diff
@@ -522,100 +534,102 @@ kubeadm init phase upload-certs --upload-certs --certificate-key=SOME_VALUE --co
...
 <!--
+### Automating kubeadm
+
 Rather than copying the token you obtained from `kubeadm init` to each node, as
 in the [basic kubeadm tutorial](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/), you can parallelize the
 token distribution for easier automation. To implement this automation, you must
 know the IP address that the control-plane node will have after it is started,
 or use a DNS name or an address of a load balancer.
 -->
+### kubeadm 自动化
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:618]:
```suggestion
### kubeadm 自动化   {#automating-kubeadm}
```

**Review by Sea-n — COMMENTED** [review]
/lgtm

**Comment by Sea-n** [comment]
Thanks for quick fix!
/lgtm

**Review by howieyuen — APPROVED** [review]
/approve

**Pull request merged**